esp/services/ws_workunits/ws_workunitsQuerySets.cpp

```diff
@@ -1172,7 +1172,7 @@ void retrieveQuerysetDetailsFromAlias(IEspContext &context, IPropertyTree *regis
         IPropertyTree *query = registry->queryPropTree(xpath);
         if (!query)
         {
-            UWARNLOG("No matching Query %s found for Alias %s", a->getId(), name);
+            UWARNLOG("No matching Query %s found for Alias %s", a ? a->getId() : "unknown", name);
```

Comment:
'a' is always going to be null the way it's currently coded.
It clears it on line 1170.

The warning should log the id. 
The code should either get the id before it's cleared,
or, a simpler/better fix IMO, is to change the getClear to a getLink, so it stays valid until end of scope.